arbitrum-docs/launch-orbit-chain/how-tos/orbit-chain-finality.md

```diff
@@ -0,0 +1,29 @@
+# Orbit Chain Confirmation and Finality
+
+## Child Chain Transactions
+
+Generally, transactions executed through the sequencer on Orbit chains [achieve finality](/tx-lifecycle) equivalent to their parent chain once the relevant transaction data has been [posted in a batch](/sequencer). This means that transactions on Orbit chains are considered final in minutes.
+
+## Parent Chain → Child Chain Transactions
+
+Messages being sent through the delayed inbox of a parent chain as [retryable tickets](/arbos/l1-to-l2-messaging#retryable-tickets), including deposits through token bridges, are released by a [well-behaved Sequencer](/inside-arbitrum-nitro/#if-the-sequencer-is-well-behaved) once they have good confidence of finality on a parent chain. On an L2 settling to Ethereum, this means that a sequencer will release delayed messages to the inbox after 40 blocks. Following this, the transaction will need to complete another finality period for the Ethereum transaction that promoted them to achieve finality.
```

Comment:
```suggestion
Messages being sent through the delayed inbox of a parent chain as [retryable tickets](/arbos/l1-to-l2-messaging.mdx#retryable-tickets), including deposits through token bridges, are released by the [sequencer](/inside-arbitrum-nitro/inside-arbitrum-nitro.mdx#if-the-sequencer-is-well-behaved) once it has good confidence of finality on the parent chain. For example, on an L2 chain settling to Ethereum, the sequencer will release delayed messages to the inbox after 40 blocks. Following this, the transaction will need to complete another finality period for the Ethereum transaction that promoted it to achieve finality.
```
Rewording it a bit to make it a bit more concise, but feel free to take and dismiss any of those changes.
(This is a non-blocking comment)